pkg/reconcileStatus/reconcileStatus.go

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
Is this actually used anywhere? It feels like it should be part of the API type and not sitting in `pkg`, that is, if it's actually being used - I couldn't immediately spot where. If not, it's safe to remove.

Reply:
This can be removed, It was a leftover from trying to remove reconcilerBase
Thanks